afs-storage-api/src/main/java/com/powsybl/afs/storage/check/FileSystemCheckIssue.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.afs.storage.check;
+
+import java.util.Objects;
+import java.util.UUID;
+
+/**
+ * @author Yichen TANG <yichen.tang at rte-france.com>
+ */
+public class FileSystemCheckIssue {
+
+    private UUID uuid;
+    private String name;
+    private Type type;
```

Comment:
We cannot use an enum here, we have to use a string, because some issues will be specific to one implementation of `AppStorage`, possibly not known to us